lib/settings/general_screen.dart

```diff
@@ -80,23 +80,10 @@ class GeneralCfgScreenState extends State<GeneralCfgScreen> {
     overrideRelease =
         config.getString('overrideRelease', category: 'developer');
     verboseLogging = config.getFlag('verboseLogging', category: 'developer');
-    selfDestructMode =
-        config.getFlag('selfDestructMode', category: 'topsecret');
-
     screenRotation = screenRotation == '' ? '0' : screenRotation;
     config.setString('screenRotation', screenRotation, category: 'advanced');
     originalRotation = screenRotation;
-  }
-
-  bool shouldDestruct() {
```

Comment:
You should bring back self destruct mode, and just gate it behind a config field. Could even be a default-off hidden/undocumented config field, like `enableEasterEggs` or something. On the whole, I don't think it's gonna be any more dicey from an other-companies standpoint than our pride-month banner.

Reply:
This and more easteregg will return after the upcoming refactor on settings and the onboarding PR ^^